### PR TITLE
embree 4.0.0, ospray 2.11.0

### PR DESCRIPTION
**Pull request by pthariensflame**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  test.c:2:10: fatal error: 'embree3/rtcore.h' file not found
  #include <embree3/rtcore.h>
           ^~~~~~~~~~~~~~~~~~
  1 error generated.
```

File: Formula/ospray.rb [hunk 1]
```diff
@@ -22,46 +22,50 @@ class Ospray < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "ispc" => :build
   depends_on "embree"
+  depends_on "ispc"
```
Comment (ZhongRuoyu) [inline Formula/ospray.rb:26]: https://github.com/Homebrew/homebrew-core/actions/runs/4372182343/jobs/7648869573#step:7:61:

```
==> brew linkage --test ospray
==> FAILED
Full linkage --test ospray output
  Broken dependencies:
    /opt/homebrew/opt/ispc/lib/libispcrt.1.dylib (ispc)
Truncated linkage --test ospray output
  Error: Broken dependencies:
    /opt/homebrew/opt/ispc/lib/libispcrt.1.dylib (ispc)
```

File: Formula/ospray.rb [hunk 2]
```diff
@@ -28,13 +28,16 @@ class Ospray < Formula
...
   end
 
   resource "openvkl" do
-    url "https://github.com/openvkl/openvkl/archive/v1.3.0.tar.gz"
-    sha256 "c6d4d40e6d232839c278b53dee1e7bd3bd239c3ccac33f49b465fc65a0692be9"
+    url "https://github.com/openvkl/openvkl/archive/refs/tags/v1.3.2.tar.gz"
+    sha256 "7704736566bf17497a3e51c067bd575316895fda96eccc682dae4aac7fb07b28"
+
+    # Fix CMake install location.
+    patch :DATA
```
[the remark this answers is not in the document]
Comment (ZhongRuoyu) [reply Formula/ospray.rb]: Submitted at https://github.com/openvkl/openvkl/pull/18.

**Comment by dawidd6** [comment]
On Linux:
```
  -- Found Embree v4.0.0: /home/linuxbrew/.linuxbrew/lib/libembree4.so.4
  CMake Error at cmake/ospray_macros.cmake:509 (message):
    Your Embree build needs to support at least one ISA >= SSE4.1 or NEON!
  Call Stack (most recent call first):
    cmake/ospray_options.cmake:90 (ospray_determine_embree_isa_support)
    CMakeLists.txt:42 (include)
```

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**